docs/game/unreal/v4.2.7.md

```diff
@@ -0,0 +1,283 @@
+   Select **Allow**
+
+   <img src={require('/img/game-mac-1.png').default} alt="blank" width="200" />
+
+   :::
+
+## STEP TWO - Create a new project
+
+1. Inside the **Unreal Project Browser**, select **Games** and click **Next**
+
+   <img src={require('/img/v4-games-2.png').default} alt="blank" width="500" />
+
+2. On the **Template** screen, select **Blank** and click **Next**
+
+    :::note
+
+    If you can't see any templates, you should reinstall v4
+
+    :::
+
+    <img src={require('/img/game-template.png').default} alt="blank" width="500" />
+
+3. In **Project Settings**
+* From the **Blueprint** and **C++** options, select **C++**. 
+* From the **Starter Content** and **No Starter Content** options, select **No Starter Content**
+* From the **Desktop/Console** or **Mobile** options, select your preference. 
+
+    <img src={require('/img/game-v4-2.png').default} alt="blank" width="500" />
+
+
+4. Give your project a memorable name e.g. BlockchainGame and then click **Create Project**
+
+5. Check your project is successfully created and opens. Then close it. So we can do the next important step of adding the SDK. 
+
+<img src={require('/img/game-mac-2.png').default} alt="create folder" width="350" />
+
+## STEP THREE - Download Ankr SDK
+
+1. Download the latest release of **AnkrSDK.zip** package from https://github.com/Ankr-network/game-unreal-sdk/releases.
+
+2. Create a new folder (if it doesn't exist) inside the project you just created and call it **Plugins**.
+
+3. Extract the downloaded **AnkrSDK.zip** contents into the **Plugins folder** you just created. 
+
+4. Delete the **AnkrSDK.zip** file if it is inside the **Plugins folder** as well. 
+
+<img src={require('/img/game-mac-3.png').default} alt="create folder" width="400" />
+
+
+## STEP FOUR - Generate the xcode project
+
+:::info 
+
+These docs are written using xcode 13.1
+
+:::
+
+1. Open **Unreal Editor**
+
+2. If you get a notice saying **"Missing BlockchainGame Modules Ankr SDK"**. Select **Yes** to rebuild.
+
+3. It should now compile the project. 
+
+4. Once the project is compiled, select **Tools** > **Generate xcode project**
+
+5. When this is done, select **Tools** > **Open xcode**
+
+## STEP FIVE - Build project
+
+1. Inside **xcode**, you should be able to see the **AnkrSDK** included in the **Plugins** folder.
+
+2. Expand **Games > your_unreal_project__name > Source > your_unreal_project_name > your_unreal_project_name.Build.cs** 
+
+   <img src={require('/img/game-mac-5.png').default} alt="xcode" width="600" />
+
+   Add `"AnkrSDK"` at the end of the string array as shown below: 
+
+   ```cpp
+   PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "AnkrSDK" });
+
+3. Now from the top menu, select **Product** > **Build For** > **running**
+
+4. You can view the Build status in the top right. 
+
+5. You know everything has worked out when a message appears stating **'Build Succeeded'**. 
```

Comment:
You know everything has worked out when a message **'Build Succeeded'** appears 
 
?